backup_and_restore/application_backup_and_restore/troubleshooting.adoc

```diff
@@ -28,6 +28,29 @@ include::modules/velero-obtaining-by-accessing-binary.adoc[leveloffset=+1]
 include::modules/oadp-debugging-oc-cli.adoc[leveloffset=+1]
 include::modules/migration-debugging-velero-resources.adoc[leveloffset=+1]
 
+[id="oadp-pod-crash-resource-request"]
+== Pods crash or restart due to lack of memory or CPU
+
+You can set specific resource requests for memory or CPU if a Velero or Restic pod crashes due to a lack of either of those resources.
```

Comment:
@RichardHoch 
I would suggest a reverse order for this sentence, first the problem, then the solution:
"If a Velero or Restic pod crashes due to a lack of memory or CPU, you can set specific requests for either of those resources."

Reply:
Nice!